package.json

```diff
@@ -47,6 +48,7 @@
     "eslint-plugin-react": "^7.17.0",
     "jest": "^24.9.0",
     "lerna": "^3.19.0",
+    "npm-publish-git": "git://github.com/hedgepigdaniel/npm-publish-git.git",
```

Comment:
what about `@hedgepigdaniel/npm-publish-git`?

Reply:
:+1: it has been done